src/index.css

```diff
@@ -14,6 +14,7 @@ code {
 
 html {
   background-image: linear-gradient(#6CD0E7, #9DEA8D);
+  background-attachment: fixed;
```

Comment:
Keeps the background from repeating when there is an overflow.

Reply:
Was it repeating somewhere? I don't remember seeing it repeat.